guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md

```diff
@@ -105,6 +105,9 @@ Sample result:
 {:.bs-callout .bs-callout-info}
 Enabling a [cache type](https://glossary.magento.com/cache-type) automatically clears that cache type.
 
+{:.bs-callout .bs-callout-info}
+Starting in version 2.3.4, you can cache user defined EAV attributes while retrieving them. By default getting all system EAV attributes is cached now. But developers can also cache user defined attributes by setting `Cache User Defined Attributes` to `Yes` in Admin Configuration or by the following CLI command: `bin/magento dev/caching/cache_user_defined_attributes 1`. Caching EAV attributes while retrieving improves performance as it decreases the amount of insert/select requests to the DB. But it increases cache network size as well.
```

Comment:
@adifucan Is the command correct? It looks like it should contain config:set

If so, then the symlink from  `guides/v2.3/config-guide/prod/config-reference-most.md` to `guides/v2.2/config-guide/prod/config-reference-most.md` will need to be broken, and the dev/caching/cache_user_defined_attributes config path added to the **Developer paths** table at the bottom of the topic.

Reply:
You right. Thanks! I'll update the PR.